backend/src/game_objects/content.rs

```diff
@@ -0,0 +1,26 @@
+use serde::Serialize;
+
+#[derive(Clone, Serialize)]
+pub enum WallKind {
+    Normal,
+}
+
+#[derive(Clone, Serialize)]
+pub struct Wall {
+    pub wall: WallKind,
```

Comment:
It seems to me that more logical name for variable here would be "kind" rather than "wall".

Reply:
yes